net/quagga/Makefile

```diff
@@ -0,0 +1,329 @@
+endef
+
+define Package/quagga-nhrpd
+  $(call Package/quagga/Default)
+  DEPENDS+=+quagga-libzebra +libcares
+  TITLE:=NHRP routing engine
+endef
+
+define Package/quagga-ospfd
+  $(call Package/quagga/Default)
+  DEPENDS+=+quagga-libospf
+  TITLE:=OSPFv2 routing engine
+endef
+
+define Package/quagga-ospf6d
+  $(call Package/quagga/Default)
+  DEPENDS+=+quagga-libospf @IPV6
+  TITLE:=OSPFv3 routing engine
+endef
+
+define Package/quagga-pimd
+  $(call Package/quagga/Default)
+  DEPENDS+=+quagga-libzebra
+  TITLE:=PIM routing engine
+endef
+
+define Package/quagga-ripd
+  $(call Package/quagga/Default)
+  DEPENDS+=+quagga-libzebra
+  TITLE:=RIP routing engine
+endef
+
+define Package/quagga-ripngd
+  $(call Package/quagga/Default)
+  DEPENDS+=+quagga-libzebra @IPV6
+  TITLE:=RIPNG routing engine
+endef
+
+define Package/quagga-vtysh
+  $(call Package/quagga/Default)
+  DEPENDS+=+quagga-libzebra +libreadline +libncurses
+  TITLE:=integrated shell for Quagga routing software
+endef
+
+define Package/quagga-zebra/conffiles
+/etc/quagga/zebra.conf
+endef
+
+define Package/quagga-bgpd/conffiles
+/etc/quagga/bgpd.conf
+endef
+
+define Package/quagga-isisd/conffiles
+/etc/quagga/isisd.conf
+endef
+
+define Package/quagga-nhrpd/conffiles
+/etc/quagga/nhrpd.conf
+endef
+
+define Package/quagga-ospfd/conffiles
+/etc/quagga/ospfd.conf
+endef
+
+define Package/quagga-ospf6d/conffiles
+/etc/quagga/ospf6d.conf
+endef
+
+define Package/quagga-pimd/conffiles
+/etc/quagga/pimd.conf
+endef
+
+define Package/quagga-ripd/conffiles
+/etc/quagga/ripd.conf
+endef
+
+define Package/quagga-ripngd/conffiles
+/etc/quagga/ripngd.conf
+endef
+
+ifneq ($(SDK),)
+CONFIG_PACKAGE_quagga-libzebra:=m
+CONFIG_PACKAGE_quagga-libospf:=m
+CONFIG_PACKAGE_quagga-watchquagga:=m
+CONFIG_PACKAGE_quagga-zebra:=m
+CONFIG_PACKAGE_quagga-bgpd:=m
+CONFIG_PACKAGE_quagga-isisd:=m
+CONFIG_PACKAGE_quagga-nhrpd:=m
+CONFIG_PACKAGE_quagga-ospf6d:=m
+CONFIG_PACKAGE_quagga-pimd:=m
+CONFIG_PACKAGE_quagga-ripd:=m
+CONFIG_PACKAGE_quagga-ripngd:=m
+CONFIG_PACKAGE_quagga-vtysh:=m
+endif
+
+CONFIGURE_ARGS+= \
+	--localstatedir=/var/run/quagga \
+	--sysconfdir=/etc/quagga/ \
+	--enable-shared \
+	--disable-static \
+	--enable-user=network \
+	--enable-group=network \
+	--enable-vty-group=network \
+	--enable-multipath=16 \
+	--disable-ospfclient \
+	--disable-capabilities \
+	--disable-doc \
+	$(call autoconf_bool,CONFIG_PACKAGE_quagga-libzebra,zebra) \
+	$(call autoconf_bool,CONFIG_PACKAGE_quagga-libospf,ospfd) \
+	$(call autoconf_bool,CONFIG_PACKAGE_quagga-bgpd,bgpd) \
+	$(call autoconf_bool,CONFIG_PACKAGE_quagga-isisd,isisd) \
+	$(call autoconf_bool,CONFIG_PACKAGE_quagga-nhrpd,nhrpd) \
+	$(call autoconf_bool,CONFIG_PACKAGE_quagga-ospf6d,ospf6d) \
+	$(call autoconf_bool,CONFIG_PACKAGE_quagga-pimd,pimd) \
+	$(call autoconf_bool,CONFIG_PACKAGE_quagga-ripd,ripd) \
+	$(call autoconf_bool,CONFIG_PACKAGE_quagga-ripngd,ripngd) \
+	$(call autoconf_bool,CONFIG_PACKAGE_quagga-vtysh,vtysh) \
+
+#MAKE_FLAGS += \
```

Comment:
Something like this is not relevant anymore.

Reply:
I'll review the makefile and the init ASAP